Coralogix/Sources/CoralogixRum.swift

```diff
@@ -233,6 +233,9 @@ public struct CoralogixExporterOptions {
     
     // A list of instruments that you wish to switch off during runtime. all instrumentations are active by default.
     var instrumentations: [InstrumentationType: Bool]?
+    
+    // Determines whether the SDK should collect the user's IP address and corresponding geolocation data. Defaults to true.
+    var skipEnrichmentWithIp: Bool = false
```

Comment:
its always confusing to present the NOT
i suggest same as browser sdk
`collectIPData: true`